hack/disconnected.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+DESTINATION_PATH=$1
```

Comment:
Can we have this fail if not specified?  E.g. `...=${1?"error message "}`

Reply:
I dont see why not. Right now its an optional var but we can force require it

Reply:
Ah - I missed it being optional - nvm then.